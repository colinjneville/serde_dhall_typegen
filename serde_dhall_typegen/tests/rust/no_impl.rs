#[serde_dhall_typegen::dhall_types("../../../tests/dhall/schema/",
    anonymous_enum_impl = false,
)]
mod dhall { }

fn main() {
    let persons = serde_dhall::from_file("../../../tests/dhall/persons.dhall").parse::<Vec<dhall::Person>>().unwrap();
    for person in persons {
        println!("{}", person.age_range.is_adult());
    }
}