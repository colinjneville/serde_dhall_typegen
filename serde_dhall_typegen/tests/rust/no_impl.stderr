error[E0599]: no method named `is_adult` found for enum `PersonAnon0` in the current scope
 --> $DIR/no_impl.rs:9:41
  |
1 | / #[serde_dhall_typegen::dhall_types("../../../tests/dhall/schema/",
2 | |     anonymous_enum_impl = false,
3 | | )]
  | |__- method `is_adult` not found for this
...
9 |           println!("{}", person.age_range.is_adult());
  |                                           ^^^^^^^^ method not found in `PersonAnon0`